### ISSUE TITLE
HOLD: OpenShift 4-10 (the final PUSH)

### ISSUE THREAD

**Issue by ryanj** [openshift-instruqt/instruqt#122]
# HOLD: Blocked by #128

1. Push updates to tracks that were automatically upgraded to 4-10 by Natale
2. File bug reports for any issues that are discovered during QA review

### Developing on OpenShift
 - [ ] [developing-on-openshift/deploying-python/](https://play.instruqt.com/openshift/tracks/deploying-applications-from-source)
   - [x] push completed
   - [ ] QA Review 
 - [ ] [developing-on-openshift/helm/](https://play.instruqt.com/openshift/tracks/developing-on-openshift-helm)
   - [ ] push completed
   - [ ] QA Review
 - [ ] developing-on-openshift/port-forwarding/
   - [ ] push completed
   - [ ] QA Review
 - [ ] developing-on-openshift/serverless/
   - [ ] push completed
   - [ ] QA Review

### Operator Framework
 - [ ] operatorframework/ansible-k8s-modules/
   - [ ] push completed
   - [ ] QA Review
 - [ ] operatorframework/ansible-mcrouter-operator/
   - [ ] push completed
   - [ ] QA Review
 - [ ] operatorframework/ansible-operator-overview/
   - [ ] push completed
   - [ ] QA Review
 - [ ] operatorframework/go-operator-memcached/
   - [ ] push completed
   - [ ] QA Review
 - [ ] operatorframework/go-operator-podset/
   - [ ] push completed
   - [ ] QA Review
 - [ ] operatorframework/helm-operator-memcached/
   - [ ] push completed
   - [ ] QA Review
 - [ ] operatorframework/helm-operator/
   - [ ] push completed
   - [ ] QA Review

### Serverless
- [ ] [serverless-camel-k-basic](https://play.instruqt.com/openshift/tracks/serverless-camel-k-basic)
   - [x] push completed
   - [ ] QA Review
- [ ] [serverless-camel-k-serving](https://play.instruqt.com/openshift/tracks/serverless-camel-k-serving)
   - [ ] push completed
   - [ ] QA Review
- [ ] [serverless-camel-k-eventing](https://play.instruqt.com/openshift/tracks/serverless-camel-k-eventing)
   - [ ] push completed
   - [ ] QA Review
- [ ] [serverless-camel-kafka-connector](https://play.instruqt.com/openshift/tracks/serverless-camel-kafka-connector)
   - [ ] push completed
   - [ ] QA Review
- [ ] [serverless-serverless](https://play.instruqt.com/openshift/tracks/serverless-serverless)
   - [ ] push completed
   - [ ] QA Review

**Comment by ryanj** [comment]
closing in favor of #161
